ci/templatetags/seed.py

```diff
@@ -2,9 +2,21 @@
 
 import seed_control_interface
 
+import re
+
 register = template.Library()
 
 
 @register.simple_tag
 def current_version():
     return seed_control_interface.__version__
+
+
+@register.filter
+def unslug(value):
+    return re.sub('[^0-9a-zA-Z]+', ' ', value).strip()
```

Comment:
Oh right, I forget, in python `_` is contained in `\w`. You can ignore this comment then.